docs/development/pigs/pig-005.rst

```diff
@@ -0,0 +1,192 @@
+.. include:: ../../references.txt
+
+.. _pig-005:
+
+***************************
+PIG 5 - Gammapy 1.0 Roadmap
+***************************
+
+* Author: Axel Donath (editor), Régis Terrier & Christoph Deil
+* Created: September 28, 2018
+* Accepted: -
+* Status: open
+* Discussion: -
+
+Introduction
+============
+This PIG describes the required short- and medium-term **development work up to
+the Gammapy 1.0** release. The anticipated time scale for this development effort
+is **9 - 12 months** and will be concluded by the Gammapy 1.0 release in fall 2019.
+The question of **API design and sub-module structure for Gammapy 1.0 will be
+addressed in a separate PIG**.
+
+The content of this document was decided based upon user feedback from the first
+CTA data challenge (DC1), experience from analysing existing datasets as well as
+definition of use cases (see below). The content will be **updated in the comming month**
+and be adjusted to upcoming **requirements defined by CTA**. The document will
+result in a series of subsequent PIGs, which are written and implemented by contributing
+developers, that **take responsibility for one of the projects** described below.
+
+
+Releases (add responsible person)
+=================================
+Up to the Gammapy 1.0 release we will pursue a **shorter release cycle**, with
+releases  **every 2-3 months**. The following releases are planned:
+
+* Gammapy 0.9 in November 2018
+* Gammapy 0.10 in January 2019
+* Gammapy 0.11 in March 2019
+* Gammapy 0.12 in Mai 2019
+* Gammapy 0.13 in Juli 2019
+* Gammapy 1.0 in October 2019
+
+The exact schedule is flexible, as it depends on when features are ready to ship.
+With this process we aim to enhance user feedback as well as set intermediate
+**milestones for the development progress**. We look for a **release manager**
```

Comment:
Remove we look for a release manager here. I'm happy to keep doing it as long as I'm around in 2019, and then someone else will continue. But like for the other points, I don't think trying to describe who does what in the roadmap document is a good way to document or coordinate that.

Reply:
Done